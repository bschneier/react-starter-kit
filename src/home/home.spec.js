import React from 'react';
import {mount} from 'enzyme';
import renderer from 'react-test-renderer';
import Home from './home';

describe('Home component', () => {
  test('Matches snapshot', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('Contains h1 with correct text', () => {
    let wrapper = mount(<Home />);
    let h1 = wrapper.find('h1');
    expect(h1).toHaveLength(1);
    expect(h1.text()).toEqual('My React App');
  });
});