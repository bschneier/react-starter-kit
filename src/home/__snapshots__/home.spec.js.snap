// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component Matches snapshot 1`] = `
<h1>
  My React App
</h1>
`;
